fluent-plugin-events/test/helper.rb

```diff
@@ -3,6 +3,54 @@
 require "fluent/test"
 require "fluent/test/driver/input"
 require "fluent/test/helpers"
+require 'mocha/test_unit'
 
 Test::Unit::TestCase.include(Fluent::Test::Helpers)
 Test::Unit::TestCase.extend(Fluent::Test::Helpers)
+
+def test_resource(name)
+  File.new("test/resources/#{name}")
+end
+
+def mock_get_events
+  Kubeclient::Client.any_instance.stubs(:public_send).with("get_events", {:as=>:raw})
+    .returns(File.read(test_resource('api_list_events_v1.json')))
+    
+end
+
+def mock_get_config_map
+  response = JSON.parse(File.read(test_resource('api_get_configmap_rv.json')))
+  Kubeclient::Client.any_instance.stubs(:public_send)
+    .with("get_config_map", "fluentd-config-resource-version", "sumologic").returns(Kubeclient::Resource.new(response))
+end
+
+def mock_watch_events
+  text = File.read(test_resource('api_watch_events_v1.txt'))
+  Kubeclient::Client.any_instance.stubs(:public_send)
+    .with("watch_events", {:as=>:raw, :field_selector=>nil, :label_selector=>nil, :namespace=>nil, :resource_version=>nil, :timeout_seconds=>360})
+    .returns(text.split(/\n+/))
+end
+
+def mock_watch_services
+  text = File.read(test_resource('api_watch_services_v1.txt'))
+  Kubeclient::Client.any_instance.stubs(:public_send)
+    .with("watch_services", {:as=>:raw, :field_selector=>nil, :label_selector=>nil, :namespace=>nil, :resource_version=>nil, :timeout_seconds=>360})
+    .returns(text.split(/\n+/))
+end
+
+def get_watch_resources_count_by_type_selector(type_selector, file_name)
+  text = File.read(test_resource(file_name))
+  objects = text.split(/\n+/).map {|line| JSON.parse(line)}
+  objects.select {|object| type_selector.any? {|type| type.casecmp?(object['type'])}}.count
+end
+
+def init_globals
+  @kubernetes_url = 'http://localhost:8080'
+  @apiVersion = 'v1'
```

Comment:
`apiVersion` was renamed to `api_version` though?

Reply:
Looks like we'll have to update this in the `enhance_k8s_metadata` test helper file as well

Reply:
Good catch. Going to fix this, as well as `enhance_k8s_metadata` test helper.

Reply:
BTW, it seems that for `enhance_k8s_metadata` test helper, `@apiVersion` or `@api_version` is no longer used there. so we can just removed it in `enhance_k8s_metadata` test helper?

Reply:
Ah, yeah looks like we can remove it 👍